conf/test.routes

```diff
@@ -1,7 +1,7 @@
-GET        /corporation-tax/identifiers/crn/:companyNumber              uk.gov.hmrc.incorporatedentityidentification.testonly.GetCtReferenceStubController.getCtReference(companyNumber: String)
+GET        /corporation-tax/identifiers/crn/:companyNumber                          uk.gov.hmrc.incorporatedentityidentification.testonly.GetCtReferenceStubController.getCtReference(companyNumber: String)
 
-GET        /api/feature-switches                                        uk.gov.hmrc.incorporatedentityidentification.featureswitch.api.controllers.FeatureSwitchApiController.getFeatureSwitches()
+GET        /api/feature-switches                                                    uk.gov.hmrc.incorporatedentityidentification.featureswitch.api.controllers.FeatureSwitchApiController.getFeatureSwitches()
 
-POST       /api/feature-switches                                        uk.gov.hmrc.incorporatedentityidentification.featureswitch.api.controllers.FeatureSwitchApiController.updateFeatureSwitches()
+POST       /api/feature-switches                                                    uk.gov.hmrc.incorporatedentityidentification.featureswitch.api.controllers.FeatureSwitchApiController.updateFeatureSwitches()
 
-POST       /cross-regime/register/VATC                                  uk.gov.hmrc.incorporatedentityidentification.testonly.RegisterWithMultipleIdentifiersStubController.registerWithMultipleIdentifiers
+POST       /cross-regime/register/GRS?grsRegime=VATC                                uk.gov.hmrc.incorporatedentityidentification.testonly.RegisterWithMultipleIdentifiersStubController.registerWithMultipleIdentifiers
```

Comment:
This didn't work for me when I tested it locally. I think because it is a query string it needs to be set up like this:
POST /cross-regime/register/GRS
uk.gov.hmrc.incorporatedentityidentification.testonly.RegisterWithMultipleIdentifiersStubController.registerWithMultipleIdentifiers(grsRegime: String)

And then in the registerWithMultipleIdentifiers I think we need it change it do a def and pass in the regime, e.g.
def registerWithMultipleIdentifiers(regime: String) ...